lib/puppet/util/windows/registry.rb

```diff
@@ -31,16 +31,24 @@ def open(name, path, mode = KEY_READ | KEY64, &block)
     end
 
     def values(subkey)
+      orig_enc = Encoding.default_internal
+      Encoding.default_internal = Encoding::UTF_8
```

Comment:
I forgot to mention that setting `Encoding.default_internal = Encoding::UTF_8` will run it through https://github.com/ruby/ruby/blob/v2_1_5/ext/win32/lib/win32/registry.rb#L894-L896 and encode it as UTF-8 handed back to us. So it's likely we could remove the force_encoding() although I would be hesitant to remove the method strictly for converting each line of the `REG_MULTI_SZ`.